cli-plugins/apps/overview.hbs.md

```diff
@@ -8,14 +8,29 @@ Tanzu Application Platform enables developers to quickly build and test applicat
 Developers can turn source code into a workload that runs in a container with a URL.
 
 A workload enables developers to choose application specifications, such as repository location, environment variables, service binding, and more.
-For more information on workload creation and management, see [Command Reference](command-reference.md).
 
 Tanzu Application Platform can support a range of workloads, including a serverless process that starts on demand, a constellation of microservices that functions as a logical application, or a small hello-world test app.
 
-## <a id='command-reference'></a>Command reference
+## <a id='envvars'> Environment variables with default values
 
-For information about available commands, see [Command Reference](command-reference.md).
+There are some environment variables that can be specified to have default values so users can execute their commands with the minimum required flags. These flags and naming convention are as follows:
 
-## <a id='usage-and-examples'></a>Usage and examples
+`--type`: `TANZU_APPS_TYPE`
+`--registry-ca-cert`: `TANZU_APPS_REGISTRY_CA_CERT`
+`--registry-password`: `TANZU_APPS_REGISTRY_PASSWORD`
+`--registry-username`: `TANZU_APPS_REGISTRY_USERNAME`
+`--registry-token`: `TANZU_APPS_REGISTRY_TOKEN`
```

Comment:
Because this PR was initially implemented a while back, please check the code to make sure this is the complete list of env vars supported and add any others.
I seem to recall we also have others (e.g. maven flags? + maybe more?)

Reply:
these are all the env vars we currently support.